src/parser/mod.rs

```diff
@@ -8735,6 +8779,14 @@ impl<'a> Parser<'a> {
         })
     }
 
+    /// Parse [Statement::Open]
```

Comment:
maybe I missed it, we seem to be lacking test cases for the open statement feature?

Reply:
It's part of `test_mssql_cursor`, but I'll make a separate test function just for `OPEN` for clarity

Reply:
Done 👍